locales/en.json

```diff
@@ -330,7 +330,7 @@
     "domainWarning": "Please confirm the domain of the website you are scanning the QR code from below",
     "access": " is requesting access to",
     "status": "Status",
-    "successMessage": "You are successfully logged in ",
+    "successMessage": "You are successfully logged in to ",
```

Comment:
do we need to update the literals for other languages as well ?

Reply:
no, this is for demo only (in english)